addons/mail/static/tests/core/message_model_tests.js

```diff
@@ -27,9 +27,9 @@ QUnit.test("Message model properties", async (assert) => {
         date: deserializeDateTime("2019-05-05 10:00:00"),
         id: 4000,
         needaction_partner_ids: [3],
-        starred_partner_ids: [3],
-        isStarred: true,
```

Comment:
This test shouldn't be changed, it highlights either cleaning (which should be done in master), or issue with code no longer supporting old data format.